client/js/components/prevent-overlapping.js

```diff
@@ -0,0 +1,42 @@
+import {nodeList, featureTest} from '../util';
+
+const preventOverlapping = (els) => {
+  if (!featureTest('position', 'sticky')) {
```

Comment:
Could we wrap this in a positive check? The logic of "if we can, do." seems easier to fathom?